### DISCUSSION TITLE
Telegram ve Discord takibine de puan veriliyor. Ama nasıl?

### DISCUSSION THREAD

**Discussion by zukenberger** [IBAX-io/go-ibax#894] [Soru Cevap - Türkçe]
Telegram ve Discord takibine de puan veriliyor. Ama nasıl?

**Comment by Skips1** [comment]
Bunu bende Yapamadım sanırım yapılmıyor Öğrenirsem yazarım kardeşim.

**Comment by ghost** [comment]
Ben alamadım
